src/db_driver.c

```diff
@@ -489,6 +489,8 @@ db_result_t *db_execute(db_stmt_t *stmt)
 
   sb_counter_inc(con->thread_id, rs->counter);
   sb_counter_add(con->thread_id, SB_CNT_CPU_UTIL_SUM, rs->cpu_util);
+  if (rs->counter == SB_CNT_GUARANTEED_CAPACITY_DROP)
```

Comment:
It seems like the standard is to always do braces, even for a single line body, as in

if (rs->counter == SB_CNT_GUARANTEED_CAPACITY_DROP)
{
    sb_counter_inc(con->thread_id, SB_CNT_ERROR);
}